### PR TITLE
SALTO-5198: Fix assets typeNames

### PR DESCRIPTION
**Pull request by idoamit8**
After conversation with Lior where we decided on the assets name, I changed them in our code. 

---

_Additional context for reviewer_
None

---
_Release Notes_: 
None

---
_User Notifications_: 
None

### CODE REVIEW

File: packages/jira-adapter/e2e_test/adapter.test.ts [hunk 1]
```diff
@@ -31,14 +31,14 @@ import { createInstances, createModifyInstances } from './instances'
...
-import { ASSESTS_SCHEMA_TYPE, ASSETS_ATTRIBUTE_TYPE, ASSETS_OBJECT_TYPE } from '../src/constants'
+import { OBJECT_SCHEMA_TYPE, OBJECT_TYPE_ATTRIBUTE_TYPE, OBJECT_TYPE_TYPE } from '../src/constants'
 
 const { awu } = collections.asynciterable
 const { replaceInstanceTypeForDeploy } = elementUtils.ducktype
 
 jest.setTimeout(600 * 1000)
 
-const excludedTypes = ['Behavior', 'Behavior__config', ASSESTS_SCHEMA_TYPE, 'AssetsSchemas', 'AssetsStatuses', 'AssetsStatus', 'AssetsObjectTypes', ASSETS_OBJECT_TYPE, ASSETS_ATTRIBUTE_TYPE]
+const excludedTypes = ['Behavior', 'Behavior__config', OBJECT_SCHEMA_TYPE, 'ObjectSchemas', 'ObjectSchemaStatuses', 'ObjectSchemaStatus', 'ObjectTypes', OBJECT_TYPE_TYPE, OBJECT_TYPE_ATTRIBUTE_TYPE]
```
Comment (shayc331) [inline packages/jira-adapter/e2e_test/adapter.test.ts:41]: please make `Behavior` and `Behavior__config` constants as well

File: packages/jira-adapter/src/config/api_config.ts [hunk 2]
```diff
@@ -2177,22 +2177,22 @@ const JSM_DUCKTYPE_TYPES: JiraDuckTypeConfig['types'] = {
...
   },
-  AssetsSchemas: {
+  ObjectSchemas: {
     request: {
       url: '/gateway/api/jsm/assets/workspace/{workspaceId}/v1/objectschema/list',
       recurseInto: [
         {
-          type: 'AssetsStatuses',
-          toField: 'assetsStatuses',
+          type: 'ObjectSchemaStatuses',
```
Comment (shayc331) [inline packages/jira-adapter/src/config/api_config.ts:2185]: please use constants

File: packages/jira-adapter/src/filters/assets/assets_object_type_path.ts [hunk 3]
```diff
@@ -19,30 +19,30 @@ import { collections } from '@salto-io/lowerdash'
...
 
-const createPaths = async (assetsObjectTypes: InstanceElement[]): Promise<void> => {
+const createPaths = async (objectTypes: InstanceElement[]): Promise<void> => {
   const graph = new DAG<InstanceElement>()
-  assetsObjectTypes.forEach(assetsObjectType => {
-    const parentFullName = assetsObjectType.value.parentObjectTypeId?.elemID.typeName === ASSESTS_SCHEMA_TYPE
-      ? undefined : assetsObjectType.value.parentObjectTypeId.elemID.name
+  objectTypes.forEach(objectType => {
+    const parentFullName = objectType.value.parentObjectTypeId?.elemID.typeName === OBJECT_SCHEMA_TYPE
+      ? undefined : objectType.value.parentObjectTypeId.elemID.name
```
Comment (shayc331) [inline packages/jira-adapter/src/filters/assets/assets_object_type_path.ts:30]: nit: 
```suggestion
      ? undefined
      : objectType.value.parentObjectTypeId.elemID.name
```

**Review by shayc331 — APPROVED** [review]
🥇

File: packages/jira-adapter/e2e_test/adapter.test.ts (hunk 1 above)
Comment (idoamit8) [reply packages/jira-adapter/e2e_test/adapter.test.ts:41]: Will remove them in the next PR

File: packages/jira-adapter/src/config/api_config.ts (hunk 2 above)
Comment (idoamit8) [reply packages/jira-adapter/src/config/api_config.ts:2185]: This is their only use

**Pull request merged**